views/index.ejs

```diff
@@ -1,49 +1,71 @@
 <%- include('./partials/head') %>
 <%- include('./partials/navigation') %>
-
+ 
 <body>
-    <main>  
-        <header>
-            <h1>Favorieten</h1>
-            <hr>
-            <p>In dit overzicht vind je al jouw favoriete huizen terug.</p>
-        </header>
-        
-        <div class="container">
-            <div class="grid-container">
-                <% alleHuizen.forEach((huis, index) => { %>
-                    <div class="card-favorites">
```

Comment:
misschien kan je van deze div een ul maken met daarin li voor de elementen in de card